templates/openstack/install.html

```diff
@@ -1,25 +1,25 @@
 {% extends "openstack/_base_openstack.html" %}
 
 
-{% block title %}Install | OpenStack | Ubuntu{% endblock %}
-{% block meta_description %}A step-by-step guide to OpenStack installation.{% endblock %}
+{% block title %}Install OpenStack yourself on Ubuntu.{% endblock %}
+{% block meta_description %}A step-by-step guide to OpenStack installation. You only need your workstation to get started.{% endblock %}
 {% block meta_copydoc %}https://docs.google.com/document/d/1eQ16jzv6YRGzG_k_C-tUnhM06k6_Ic6fZM-x_ARMzuc/edit{% endblock meta_copydoc %}
 
 {% block content %}
 <section class="p-strip--suru-image is-deep is-bordered">
   <div class="row u-equal-height u-vertically-center ">
     <div class="col-7">
       <h1>Install OpenStack yourself</h1>
-      <p>Test OpenStack on your workstation, set up an edge cloud or deploy a production-grade cloud across hundreds of physical servers in a data centre. You will use <a href="https://microstack.run/" class="p-link--external">MicroStack</a> or <a href="https://jaas.ai/openstack" class="p-link--external">Charmed OpenStack</a> depending on the requirements.</p>
-      <p>If you run into issues, or if you want support, training or architecture design consulting, please contact Canonical &mdash; we help the world's largest OpenStack users keep their clouds running smoothly.</p>
+      <p>Try OpenStack on your workstation, set up a micro cloud or deploy a production-grade OpenStack cluster across hundreds of physical machines. You will use <a href="https://microstack.run/" class="p-link--external">MicroStack</a> or <a href="/openstack">OpenStack Charms</a> depending on the requirements.</p>
+      <p>If you run into any issues, or if you want support, OpenStack consulting, support or fully managed service, speak to an expert.</p>
       <p>
-        <a class="p-button--positive js-invoke-modal" href="/openstack/contact-us">
+        <a class="p-button--positive js-invoke-modal" href="/openstack/#get-in-touch">
```

Comment:
```suggestion
        <a class="p-button--positive js-invoke-modal" href="/openstack/contact-us?product=openstack">
```
When JS is disabled this button redirects to another page where there is a button that redirects to a static form. It might be easier to redirect straight to this form.